src/PrestaShopBundle/Form/Admin/Sell/Category/DeleteCategoriesType.php

```diff
@@ -31,6 +31,7 @@
 use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 use Symfony\Component\Form\Extension\Core\Type\CollectionType;
 use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
```

Comment:
@PululuK Could you remove it, please ? It's not used.

Reply:
Hi @Progi1984 
Sorry, I'm getting old and I'm going blind 😅
Done
Thank you